src/jmp.zig

```diff
@@ -1,3 +1,8 @@
-pub const jmp = @cImport({
+// FIXME: remove setjmp/longjmp  https://github.com/ziglang/zig/issues/1656#issuecomment-840190292
+// tip: use minicoro instead  https://github.com/edubart/minicoro
```

Comment:
Using something like minicoro is planned anyway: right now fibers are not jit compiled with mir because I have no easy way to switch context across mir/buzz call frames.